dist/index.js

```diff
@@ -32,6 +37,10 @@ function scanCLI(cliArgs, params, callback) {
   })
 }
 
+if (foo == '') {
+  
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the useless trailing whitespaces at the end of this line. ([javascript:TrailingWhitespace](https://sonarcloud.io/organizations/default/rules#rule_key=javascript%3ATrailingWhitespace))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonarqube-scanner&pullRequest=38&issues=AWLNyHlbjr7wspRoiHiS&open=AWLNyHlbjr7wspRoiHiS)

[ISSUE_KEY]: AWLNyHlbjr7wspRoiHiS